roadmap/implementers-guide/src/node/utility/availability-store.md

```diff
@@ -193,7 +197,7 @@ Basically we need to test the correctness of data flow through state FSMs descri
   - Wait until the data should have been pruned.
   - The data is no longer available.
 
-- Forkfulness of the relay chain is taken into account
+- Fork-awareness of the relay chain is taken into account
```

Comment:
I don't think they mean the same, `forkfulness` is the correct term here

Reply:
The _implementation_ is `fork-aware`, so this doesn't express the same thing indeed, forkfulness is still a mouthful and not very clear.